tools/jnimarshalmethod-gen/App.cs

```diff
@@ -39,6 +58,9 @@ public static int Main (string [] args)
 				{ "d|debug",
 				  "Inject debug messages",
 				  v => Debug = true },
+				{ "k|keep-temporary",
```

Comment:
This should be `keeptemp`, as that's what [**mkbundle**](http://docs.go-mono.com/?link=man%3amkbundle(1)) uses.

I also don't think it needs a "short" `-k` form.